src/components/UserMenuListItem/__tests__/UserMenuListItem.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+import UserMenuListItem from 'components/UserMenuListItem';
+import { Newbie } from 'buddy-app-schema';
+import { mockSchemaTypes } from '__mocks__';
+
+jest.mock('@material-ui/core/ListItem', () => 'ListItem');
+jest.mock('@material-ui/core/Box', () => 'Box');
+jest.mock('@material-ui/core/Typography', () => 'Typography');
+jest.mock('components/Avatar', () => 'Avatar');
+jest.mock('buddy-app-schema', () => mockSchemaTypes());
```

Comment:
Since [PR-39](https://github.com/BuildIt-Poland/buddy-app/pull/39) is merged this mock is unnecessary